daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/PackageInterface.scala

```diff
@@ -418,6 +418,28 @@ private[lf] class PackageInterface(signatures: PartialFunction[PackageId, Packag
   val packageLanguageVersion: PartialFunction[PackageId, LanguageVersion] =
     signatures andThen (_.languageVersion)
 
+  def interfaceImplementations(
```

Comment:
Consider adding some docs to make it clear what the return type represents. TypeConName for both interfaces & templates make the type not all that descriptive. I think it would also be good to explicitly state that this mapping only includes interface implementations in the given package id not in dependencies of that.